### PR TITLE
Allow metrics to calculate the sum or average. Closes #517.

### LINKED ISSUE #517: Metrics adding up instead of averaging
I am trying to set up so that metrics are pushed every second to the server (ping). The server is just adding them up instead of averaging them out.

Image of graph area
Image of points

### PR DESCRIPTION
**Pull request by jbrooksuk**
Also fixes a bug found in #515 where only the last graph would render.

### CODE REVIEW

**Comment by jbrooksuk** [comment]
@joecohens this is also backwards compatible with existing metrics :+1:

**Comment by joecohens** [comment]
Sweet! :+1:

File: app/views/dashboard/metrics/add.blade.php [hunk 1]
```diff
@@ -32,6 +32,13 @@
                             </div>
                         </div>
                         <div class="form-group">
+                            <label>{{ trans('forms.metrics.type') }}</label>
```
Comment (joecohens) [inline app/views/dashboard/metrics/add.blade.php]: Should be `forms.metrics.calc_type` or we need to change the lang key :)

File: app/views/dashboard/metrics/edit.blade.php [hunk 2]
```diff
@@ -32,6 +32,13 @@
                             </div>
                         </div>
                         <div class="form-group">
+                            <label>{{ trans('forms.metrics.type') }}</label>
```
Comment (joecohens) [inline app/views/dashboard/metrics/edit.blade.php]: Same here.

File: app/views/dashboard/metrics/add.blade.php (hunk 1 above)
Comment (jbrooksuk) [inline app/views/dashboard/metrics/add.blade.php]: Whoops!

File: app/views/dashboard/metrics/edit.blade.php (hunk 2 above)
Comment (jbrooksuk) [inline app/views/dashboard/metrics/edit.blade.php]: This is a weird change.

**Comment by jbrooksuk** [comment]
All done :)

**Pull request merged**